devolutions-gateway/src/api/jrec.rs

```diff
@@ -376,6 +397,11 @@ pub(crate) async fn list_recordings(
     }
 }
 
+async fn list_active_recordings(recordings: RecordingMessageSender) -> Result<Json<Vec<Uuid>>, HttpError> {
+    let recordings = recordings.active_recordings.copy_set().into_iter().collect();
+    Ok(Json(recordings))
+}
```

Comment:
issue: This function is not performing any async operation.

suggestion: Inline the function as it’s reused nowhere and is only a very readable single liner.

Reply:
updated